### PR TITLE
eLife citation

### PR DESCRIPTION
**Pull request by David-Peede**
The BibTeX citation from the elife website was a tad different than the original bioRxiv citation. I also removed the "." after the authors' middle initial to reflect elife's author list style (see https://elifesciences.org/articles/54967).

### CODE REVIEW

**Comment by andrewkern** [comment]
great thanks @David-Peede! as we merge code PRs get a bit behind, so to maintain a clean commit history we ask contributors to `rebase` if this happens. Would you mind rebasing following [these instructions](https://stdpopsim.readthedocs.io/en/latest/development.html#rebasing) and then this looks good to merge

**Comment by David-Peede** [comment]
Rebased! Do I need to do anything for merging or is that done by a reviewer?

**Comment by andrewkern** [comment]
so generally what happens is that you request reviewers or they are assigned to you. in this case I assigned myself to review. Reviews can then comment on the code, make suggestions, or request changes. Once the code is approved Maintainers of the repo are responsible for the merge. I'm gonna take care of that in this case too 😄

**Review by andrewkern — APPROVED** [review]

**Pull request merged**
